core/src/services/dropbox/core.rs

```diff
@@ -111,6 +111,23 @@ impl DropboxCore {
         self.client.send(request).await
     }
 
+    pub async fn dropbox_get_stat(&self, path: &str) -> Result<Response<IncomingAsyncBody>> {
+        let url = "https://api.dropboxapi.com/2/files/get_metadata".to_string();
```

Comment:
Maybe we need to use const define `DROPBOX_METADATA_URL`?

Reply:
> Maybe we need to use const define `DROPBOX_METADATA_URL`?

Current style is Ok to me. We don't need to extract const so early.